cfeScraping/index.js

```diff
@@ -41,4 +41,18 @@ ScrapPage.prototype.fillInput = async function (id, text) {
   }
 };
 
+// eslint-disable-next-line space-before-function-paren
+ScrapPage.prototype.clickButton = async function (id) {
+  try {
+    await this.page.click(id);
+    console.log('Searching...');
+    await this.page.waitForTimeout(2000);
```

Comment:
you can configure the time with another param of the function or a default param too, 
`function (id, waitingTime )`
`function (id, waitingTime = 2000)`